### ISSUE TITLE
Debug css for user login

### ISSUE THREAD

**Issue by mMoliere** [majlis-erc/manuForma#331]

**Comment by mMoliere** [comment]
@wsalesky The labels in the user login are not legible, as they are white. I tried to correct the .css, but I discovered that the relevant file is not accessible to me. The inspector revealed the following location: https://manuforma-staging.jalit.org/exist/apps/manuForma/$nav-base/resources/bootstrap/css/bootstrap.min.css

The repository does no contain a folder called "$nav-base", so I guess that there is some bootstrap mechanism that I'm not familiar with.

Long story short: please make the white labels black.

**Comment by wsalesky** [comment]
See fix in https://github.com/majlis-erc/manuForma/pull/344

Changes to styles can be made in: /manuForma/resources/styles.css
this will override default bootstrap styles. The $nav-base is an application specific thing telling the app where to find the application 'home',  you can ignore and just use the path after the nav-base to find the relevant files. 

I recommend making all changes at the bottom, so as not to interfere with the styles created for the new design.

**Comment by mMoliere** [comment]
Works like a charm. Thanks for the infos!
